gems/pending/appliance_console/database_maintenance.rb

```diff
@@ -0,0 +1,51 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'fileutils'
+
+module ApplianceConsole
+  class DatabaseMaintenance
+    include ApplianceConsole::Logging
+
+    HOURLY_SEED = "/var/www/miq/vmdb/gems/pending/appliance_console/miq-pg-maintenance-hourly.cron".freeze
+    HOURLY_CRON = "/etc/cron.hourly/miq-pg-maintenance-hourly.cron".freeze
+
+    attr_accessor :frequency, :day_number, :hour_number
+
+    def ask_questions
+      clear_screen
+      confirm
+    end
+
+    def activate
+      say("Configuring Database Maintenance...")
+      configure_hourly
+    end
+
+    def confirm
+      agree("Configure Database Maintenance? (Y/N): ")
+    end
+
+    private
+
+    def configure_hourly
+      if hourly_configured?
+        deactivate_hourly
+      else
+        say("Configuring Hourly Database Maintenance...")
+        FileUtils.cp(HOURLY_SEED, HOURLY_CRON)
+        FileUtils.chmod(0755, HOURLY_CRON)
+        true
+      end
+    end
+
+    def deactivate_hourly
+      return true unless agree("\Hourly Database Maintenance is already configured, Un-Configure (Y/N):")
+      File.delete(HOURLY_CRON)
```

Comment:
I think `File.delete` would probably raise an error if the file wasn't there or if there was a permissions prompt or something like that where `FileUtils.rm_f` would not (similar to `rm -f` vs `rm`)